hardhat.config.js

```diff
@@ -45,6 +45,16 @@ const compilerSettings = {
         optimizer: optimizerSettings,
     },
 };
+const itsCompilerService = {
+    version: '0.8.21',
```

Comment:
```suggestion
    version: '0.8.27',
```
use a newer version?